src/article-editor.jsx

```diff
@@ -66,7 +67,7 @@ var ArticleEditor = React.createClass({
 
     render: function() {
 
-        return <div className="framework-perseus perseus-article">
+        return <div id="perseus" className="framework-perseus perseus-article">
```

Comment:
This adds a margin around the ArticleEditor that matches the margin around the EditorPage.